### PR TITLE
added Air quality and imperial quantities

### PR DESCRIPTION
**Pull request by Rocketct**

### CODE REVIEW

File: src/EnvClass.cpp [hunk 1]
```diff
@@ -102,7 +102,13 @@ float EnvClass::readTemperature(int units /*= CELSIUS*/)
 {
   if (_revision == BOARD_REVISION_2) {
     while(!iaqSensor->run()){ }
-    return iaqSensor->temperature;
+    float reading = iaqSensor->temperature;
+    if (units == FAHRENHEIT){
+      reading = iaqSensor->temperature;
```
Comment (facchinm) [inline src/EnvClass.cpp:107]: Can remove this line

File: src/PressureClass.cpp [hunk 2]
```diff
@@ -101,16 +101,29 @@ float PressureClass::readPressure(int units)
...
 
-float PressureClass::readTemperature()
+float PressureClass::readTemperature(int units /*= CELSIUS*/)
 {
   if (_revision == BOARD_REVISION_2) {
     while(!iaqSensor->run()){}
-    return iaqSensor->temperature;
+    float reading = iaqSensor->temperature;
+    if (units == FAHRENHEIT){
+      reading = iaqSensor->temperature;
```
Comment (facchinm) [inline src/PressureClass.cpp:122]: can remove this too

**Review by facchinm — CHANGES_REQUESTED** [review]

File: src/PressureClass.cpp (hunk 2 above)
Comment (Rocketct) [reply src/PressureClass.cpp]: done

File: src/EnvClass.cpp (hunk 1 above)
Comment (Rocketct) [reply src/EnvClass.cpp]: done

File: src/AirQualityClass.cpp [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#include "AirQualityClass.h"
+
+// sets function called on slave write
+AirQualityClass::AirQualityClass( getRev_t getRevision)
+{
+  //If board_revision = 1, IMU module is LSM6DSOX, otherwise is LSM6DS3
```
Comment (facchinm) [inline src/AirQualityClass.cpp:25]: Need to fix some stray comment

File: src/AirQualityClass.cpp [hunk 4]
```diff
@@ -0,0 +1,148 @@
+/*
+  This file is part of the Arduino_LSM6DSOX library.
```
Comment (facchinm) [inline src/AirQualityClass.cpp:2]: Change this to the correct name

**Review by facchinm — CHANGES_REQUESTED** [review]

File: src/AirQualityClass.cpp (hunk 3 above)
Comment (Rocketct) [reply src/AirQualityClass.cpp]: removed make sense only in IMUCLass

File: src/AirQualityClass.cpp (hunk 4 above)
Comment (Rocketct) [reply src/AirQualityClass.cpp]: changed also in IMUCLass :)

File: src/AirQaulityClass.cpp [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#include "AirQaulityClass.h"
+
+// sets function called on slave write
+AirQaulityClass::AirQaulityClass( getRev_t getRevision)
```
[the remark this answers is not in the document]
Comment (Rocketct) [reply src/AirQaulityClass.cpp]: fixed

**Review by facchinm — APPROVED** [review]

**Pull request merged**